### ISSUE TITLE
Certain apps will not resize anymore

### ISSUE THREAD

**Issue by jessehs** [eczarny/spectacle#890]
Slack, Signal Desktop, Console, are 3 examples of apps that stopped resizing. I found these error messages in the Console app:

default	09:30:55.295446 -0700	Spectacle	Unable to obtain the accessibility element with the specified attribute: AXFocusedWindow (error code -25211)
default	09:30:55.295493 -0700	Spectacle	Invalid accessibility element provided, unable to determine the size and position of the window.
